code/modules/antagonists/zombie/abilities/infect.dm

```diff
@@ -0,0 +1,89 @@
+/datum/targetable/zombie/infect
+	name = "Infect"
+	desc = "After a short delay, infect a human. If they are damaged enough or dead this will convert them instantly."
+	icon = 'icons/mob/critter_ui.dmi'
+	icon_state = "critter_bite"
+	cooldown = 0
+	var/actual_cooldown = 200
```

Comment:
Should document why this exists (i assume so the action doesn't go on cooldown if the action doesn't complete?)

Reply:
This was just copied over from before but yeah thats the use since the ability uses an action bar, theres probably a better way and this isn't the only place var is used its also used on most actions with action bars